seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java

```diff
@@ -153,248 +143,6 @@ public void testSystemMonitoringInformation() {
                         });
     }
 
-    @Test
```

Comment:
why remove this?

Reply:
> why remove this?

Migrated to ClusterSeaTunnelContainer.java

Reply:
> > why remove this?
> 
> Migrated to ClusterSeaTunnelContainer.java

Use two jvm processes to simulate real scenarios.

Reply:
oh I see.